inc/ajax.php

```diff
@@ -18,7 +18,7 @@ function sell_media_add_to_cart() {
 	global $sm_cart;
 
 	if( !isset( $_POST['_wpnonce'] ) || isset( $_POST['_wpnonce'] ) && !wp_verify_nonce( $_POST['_wpnonce'], 'sell_media_add_cart_action') ) {
-		echo 0;
+		echo esc_html( '0' );
```

Comment:
@oleg-shumar Should we be using `wp_die()` here to replace the `echo` and `exit`?

https://wordpress.stackexchange.com/questions/250280/best-way-to-end-wordpress-ajax-request-and-why

Reply:
I will comment this privately